addons/payment/views/payment_provider_views.xml

```diff
@@ -108,7 +99,7 @@
                                                widget="many2many_tags"
                                                placeholder="Select currencies. Leave empty not to restrict any."
                                                options="{'no_create': True}"
-                                               required="require_currency and state != 'disabled'"/>
+                                               required="state != 'disabled'"/>
```

Comment:
This is not equivalent to the previous logic. Now, all providers require this field to be set as soon as they are no longer in the "disabled" state, whereas it should only be AsiaPay that requires that.